roles/sp/tasks/nexus.yml

```diff
@@ -15,6 +15,6 @@
         aliases:
           - son-nexus
     published_ports:
-      - "8081:8081"
-      - "8082:8082"
-      - "8083:8083"
\ No newline at end of file
+      - "8081:8091"
```

Comment:
@efotopoulou should be the other way around, the overlapped ports are the public ones so in this case are located in the left hand. We need to invert the order `8091:8081`